src/utils.ts

```diff
@@ -232,8 +232,8 @@ export const getEnvironment = async (
       : "feature";
   Logger.info(`environmentType: ${environmentType}`);
   Logger.info(`CONTENTFUL_ALIAS: ${CONTENTFUL_ALIAS}`);
-  const isEnvTypeAlias = environmentType === CONTENTFUL_ALIAS
-  Logger.info(`isEnvTypeAlias: ${isEnvTypeAlias}`);
+  // const isEnvTypeAlias = environmentType === CONTENTFUL_ALIAS
```

Comment:
should we remove these?

Reply:
Yup, just didn't want to do it before review